### PR TITLE
Fix a couple dev dependency issues

### PR DESCRIPTION
**Pull request by ryantrem**
1. Switch from rollup-plugin-minify-html-literals (old, dependency on rollup is out of date) to rollup-plugin-minify-template-literals (newer, forked from the former, fixes this issue and a few others).
2. Add explicit dependency on @types/eslint to packages/tools/eslintBabylonPlugin/package.json (no idea why the dependency is no longer being implicitly installed, but having an explicit dependency is still correct).

### CODE REVIEW

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/tools/eslintBabylonPlugin/package.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
         "@microsoft/tsdoc-config": "~0.16.1"
     },
     "devDependencies": {
+        "@types/eslint": "^9.6.1"
```
Comment (RaananW) [inline packages/tools/eslintBabylonPlugin/package.json:27]: nit - we are using eslint 8.X, better to have this:

```suggestion
        "@types/eslint": "^8.56.10"
```

**Review by RaananW — APPROVED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15525/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15525/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15525/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15525/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15525/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15525/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15525/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15525/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15525/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15525/merge/testResults/webgpuplaywright/index.html

**Pull request merged**